src/bindings/python/BUILDING.md

```diff
@@ -1,88 +1,145 @@
-# Building the nGraph Python* API
+# Building the OpenVINO™ Python* API
 
-This document provides the instructions for building the nGraph Python API from source on Linux, macOS and Windows 10 platforms.
+This document provides the instructions for building the OpenVINO™ Python API from source on Linux, macOS and Windows 10 platforms.
 
 For each platform, you can build and install the API as a part of OpenVINO™ Toolkit or as a Python wheel.
-A Python wheel is a portable package that allows you to install nGraph in your Python distribution, or dedicated virtual environment.
+A Python wheel is a portable package that allows you to install OpenVINO™ in your Python distribution, or dedicated virtual environment.
 
 ## Linux* and macOS*
 
 ### Prerequisites
 
-To build the nGraph Python API, you need to install a few additional packages.
+To build the OpenVINO™ Python API, you need to install a few additional packages.
 
 On Ubuntu* 20.04 LTS you can use the following instructions to install the required packages, including Python and Cython.
 
-    apt install git wget build-essential cmake
-    apt install python3 python3-dev python3-pip python3-virtualenv python-is-python3
+```bash
+apt install git wget build-essential cmake
+apt install python3 python3-dev python3-pip python3-virtualenv python-is-python3
+```
 
 On macOS, you can use [Homebrew](https://brew.sh) to install required packages:
 
-    brew install cmake
-    brew install automake
-    brew install libtool
-    brew install python3
+```bash
+brew install cmake
+brew install automake
+brew install libtool
+brew install python3
+```
 
 Install Cython in the Python installation, or virtualenv that you are planning to use:
 
-    pip3 install cython
+```bash
+pip3 install cython
```

Comment:
why not to refer to generic requirements.txt files?

Reply:
Just because it is not there and never was. It is only required for building -- to just run OV or run tests it is not required.

I might add `requirements_build` if deemed necessary -- however this is only one package.